src/test/java/posters/pageobjects/pages/checkout/GuestBillingAddressPage.java

```diff
@@ -31,74 +31,79 @@ public GuestBillingAddressPage isExpectedPage()
-        $("#bill h3").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.3.name"))).shouldBe(visible);
-        $("#payment h3").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.4.name"))).shouldBe(visible);
-        $("#chkout h3").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.5.name"))).shouldBe(visible);
-        $("#orderCmplt h3").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.6.name"))).shouldBe(visible);
+        $(".progress-step-1 .progress-bubble-caption").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.1.name"))).shouldBe(visible);
+        $(".progress-step-2 .progress-bubble-caption").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.2.name"))).shouldBe(visible);
+        $(".progress-step-3 .progress-bubble-caption").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.3.name"))).shouldBe(visible);
+        $(".progress-step-4 .progress-bubble-caption").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.4.name"))).shouldBe(visible);
+        $(".progress-step-5 .progress-bubble-caption").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.5.name"))).shouldBe(visible);
+        $(".progress-step-6 .progress-bubble-caption").shouldHave(exactText(Neodymium.localizedText("AddressPages.processWrap.6.name"))).shouldBe(visible);
     }
     
     private void validateFillInHeadlines(String headline)
     {
-        $$(".form-group").findBy(exactText(headline)).shouldBe(visible);
+        $$(".mb-3").findBy(exactText(headline)).shouldBe(visible);
     }
-    
+
+    private void validateFillInHeadlinesStateZip(String headline)
+    {
+        $$(".mb-3 .col .form-label").findBy(exactText(headline)).shouldBe(visible);
+    }
+
     @Step("validate fill-in form headlines")
     public void validateFillInHeadlines()
     {
         validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.fullName"));
         validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.company"));
         validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.address"));
         validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.city"));
-        validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.state"));
-        validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.zip"));
+        validateFillInHeadlinesStateZip(Neodymium.localizedText("AddressPages.fillIn.headlines.state"));
+        validateFillInHeadlinesStateZip(Neodymium.localizedText("AddressPages.fillIn.headlines.zip"));
         // TODO - fix after issue is fixed
         //validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.country"));
```

Comment:
Is this still defect or can it a) be removed or b) taken in?

Reply:
`//validateFillInHeadlines(Neodymium.localizedText("AddressPages.fillIn.headlines.country"));`
is still a defect. 
`validateFillInHeadlinesStateZip`() works as it should.